ccan/list/list.h

```diff
@@ -126,6 +126,7 @@ struct list_node *list_check_node(const struct list_node *n,
  * Example:
  *	static LIST_HEAD(my_global_list);
  */
+#undef LIST_HEAD
```

Comment:
#include <sys/queue.h>          /* get TAILQ macros */
"/usr/include/net/if_var.h" [readonly] 235L, 9394C

if.h     108 #include <net/if_var.h>

It comes from if.h which includes if_var.h